README.md

```diff
@@ -53,7 +53,14 @@ Find out how to [update with npm](docs/installation/updating-with-npm.md).
 
 ## Licence
 
-Unless stated otherwise, the codebase is released under the MIT License. This
-covers both the codebase and any sample code in the documentation. The
-documentation is &copy; Crown copyright and available under the terms of the
-Open Government 3.0 licence.
+Unless stated otherwise, the codebase is released under [the MIT License][mit].
+This covers both the codebase and any sample code in the documentation.
+
+The documentation is [&copy; Crown copyright][copyright] and available under the terms
+of the [Open Government 3.0][ogl] licence.
+
+[rvm]: https://www.ruby-lang.org/en/documentation/installation/#managers
+[bundler]: http://bundler.io/
```

Comment:
Don't think we're using ruby (yet) 😆

Reply:
ARGH I definitely removed that before committing, I'm sure I did 😹